retsinformation.js

```diff
@@ -0,0 +1,182 @@
+	newItem.url = url;
+	if (type=="statute") {
+		newItem.nameOfAct = newItem.title
+		newItem.codeNumber = getKortNavn(doc, url)[0];
+		newItem.publicLawNumber = newItem.codeNumber;
+		newItem.dateEnacted = getKortNavn(doc, url)[1];
+		newItem.shortTitle = getRessort(doc, url)[0];
+		if (/L(BK|OV)/.test(newItem.publicLawNumber.substr(0,3))) {
+			Zotero.debug("LOV eller LBK");
+		} else {
+			newItem = addAuthor (newItem, getRessort(doc, url)[1]);
+		};
+	} else if (type=="case") {
+		newItem.caseName = newItem.title
+		newItem = addAuthor (newItem, getRessort(doc, url)[1]);		
+		newItem.docketNumber = getKortNavn(doc, url)[0];
+		newItem.dateDecided = getKortNavn(doc, url)[1];
+		newItem.shortTitle = getRessort(doc, url)[0];
+	} else if (type=="report") {
+		newItem = addAuthor (newItem, getRessort(doc, url)[1]);
+		newItem.date = getRessort(doc, url)[2];
+		newItem.institution = "Folketinget";
+		threeLetters = getKortNavn(doc, url)[0].substr(0,3);
+		if (threeLetters=="EDP") {
+			newItem.seriesTitle = "Folketingets Ombudsmands driftundersøgelser";
+		} else if (threeLetters=="ISP") {
+			newItem.seriesTitle = "Folketingets Ombudsmands Inspektioner";
+		} else if (threeLetters=="FOU") {
+			newItem.seriesTitle = "Folketingets Ombudsmands udtalelser";
+		};
+		newItem.reportNumber = getKortNavn(doc, url)[0];
+	} else if (type=="bill") {
+		newItem = addAuthor (newItem, getRessort(doc, url)[1]);
+		newItem.billNumber = getKortNavn(doc, url)[0];
+		newItem.date = getRessort(doc, url)[2];
+	} else {
+		return;
+	};
+	newItem.complete();
+	return;
+}
+
+function getTitle (doc, url) {
+	var myXPath = '//div[@class="wrapper2"]/div/p[@class="Titel2"]';
+	title = ZU.xpathText(doc, myXPath);
+	if (!title) {
+		myXPath = '//div[@class="wrapper2"]/div/div[@id="INDHOLD"]/p[@class="Titel"]';
+		title = ZU.xpathText(doc, myXPath);
+	};
+	if (!title) {
+		myXPath = '//div[@class="wrapper2"]/div/div[@id="INDHOLD"]/font/p[@align="CENTER"]';
+		title = ZU.xpathText(doc, myXPath);
+	};
+	if (!title) {
+		myXPath = '//div[@class="wrapper2"]/div/div[@id="INDHOLD"]/h1[@class="TITLE"]';
+		title = ZU.xpathText(doc, myXPath);
+	};
+	if (!title) {
+		myXPath = '//div[@class="wrapper2"]/div/div[@id="INDHOLD"]/center/h1[@class="TITLE"]';
+		title = ZU.xpathText(doc, myXPath);
+	};
+	if (!title) {
+		var myXPath = '/html/head/title';
+		title = ZU.xpathText(doc, myXPath);
+		title = title.substring(0,title.lastIndexOf("-"));
+		if (getRessort(doc, url)[0]) {
+			title = title.substring(getRessort(doc, url)[0].length+4,title.length);
+		};
+	};
+	title = clean(title);
+	Z.debug("L111"+title);
+	return title;
+}
+
+function getKortNavn (doc, url) {
+	var myXPath = '//span[@class="kortNavn"]';
+	if (!ZU.xpathText(doc, myXPath)) {
+		myXPath = '//span[@class="kortNavn historisk"]';
+	};
+	infoArray = ZU.xpathText(doc, myXPath);
+	infoArray = infoArray.replace(/ (Gældende|Historisk)/,"");
+	infoArray = infoArray.split(" af ");
+	return infoArray;
```

Comment:
it would be more readable (above) if the values were assigned to an object with appropriate properties (e.g. `.date`) and the object returned.